src/zc/FileStorage/__init__.py

```diff
@@ -86,46 +87,50 @@ def __init__(self, storage, stop,
 
     def pack(self):
 
-        script = self._name+'.packscript'
-        open(script, 'w').write(pack_script_template % dict(
-            path = self._name,
-            stop = self._stop,
-            size = self.file_end,
-            syspath = sys.path,
-            blob_dir = self.storage.blob_dir,
-            sleep = self.sleep,
-            transform = self.transform_option,
-            untransform = self.untransform_option,
-            ))
-        for name in 'error', 'log':
-            name = self._name+'.pack'+name
+        script = self._name + ".packscript"
+        open(script, "w").write(
```

Comment:
(Out of scope for this PR, but all these open()s ought to be embedded in `with` statements.)